packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -7139,19 +7141,19 @@ describe('plugin-meetings', () => {
           it('sends client.ice.start correctly when hasMediaConnectionConnectedAtLeastOnce = true', () => {
             meeting.hasMediaConnectionConnectedAtLeastOnce = true;
             meeting.setupMediaConnectionListeners();
-            eventListeners[Event.CONNECTION_STATE_CHANGED]({
-              state: 'Connecting',
-            });
+
+            meeting.mediaProperties.webrtcMediaConnection.getConnectionState = sinon.stub().returns(ConnectionState.Connecting);
```

Comment:
I don't really like the idea how it's done now. However, i have no better idea for now. I am open to any suggestions.

Reply:
instead of triggering the listeners from webrtcMediaConnection we could just stub the whole ConnectionStateHandler in these tests and trigger the events from it, then `meeting.mediaProperties.webrtcMediaConnection.getConnectionState()` would never be called, but I'm ok with your current approach too - it's a bit more messy and difficult to follow what's going on, but it tests more things as it's using the real ConnectionStateHandler. Maybe just worth wrapping the 2 calls into a helper function something like `simulateConnectionStateChange(newConnectionState)` ?

Reply:
Done.